Create LIT14520QeneZawahabkani.xml

CarstenHoffmannMarburg opened the pull request:
I created record for a Qene poem, which Denis has directed me to. However, the creation of the ID through the app ended up with the unhappy number of LIT14520, what is beyond the expected number in the 7300s. I suggest to change the number, before the branch may be eventually merged.

I did not find an authority file for the Qene-type Za-ʾamlākiya and pronounced the wish to create one in https://github.com/BetaMasaheft/Documentation/issues/2504 and  https://github.com/BetaMasaheft/Documentation/issues/2830.

For the end of the poem, there is one more phrase, which I think, does not belong to the poem (ተፈጸመ፡ ግብረ፡ ሰዓታት።), but may be some instruction or indication of the end of the service or something similar. I did not include this line to the edition of the text.

![grafik](https://github.com/user-attachments/assets/29f0581e-cec4-4535-9c9b-3a579e1cecde)

karljonaskarlsson commented on new/LIT14520QeneZawahabkani.xml:
```diff
@@ -0,0 +1,67 @@
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="EMIP00499"/>, fol. 160b. Division into verses is indicated with ። .</note>
+                <ab>
+                    <l n="1">ዘአምላኪየ፡ ዘወሀብከኒ፡ ግብረ፡ አምጣነ፡ በትጋህ፡ ፈጸምኩ።</l> 
```
Should the qene-type indication really be part of the edition?

CarstenHoffmannMarburg replied:
I am not sure, as well. For LIT7275QeneWaEsagged, we did include the indication of the qene-type to the edition in the record for the reason, that we were at first not clear about the meaning of ዋ።. However, in this case, we can be certain, that it is a type indication. 
I think I should delete in both cases the type indication. It will still be mentioned in the abstract.

DenisNosnitsin1970 replied:
ተፈጸመ፡ ግብረ፡ ሰዓታት። is not a part of qene, it is a "micro-colophon". It is in fact an interesting and complex case.

karljonaskarlsson commented on new/LIT14520QeneZawahabkani.xml:
```diff
@@ -0,0 +1,67 @@
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዘወሀብከኒ፡ ግብረ፡ አምጣነ፡ በትጋህ፡ ፈጸምኩ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Za-wahabkani gǝbra ʾamṭāna ba-tǝgāh faṣṣamku ... (qǝne of the zaʾamlākiya-type)</title>
```
I would transcribe "Za-wahabkanni"

And I think it has to be "za-ʾamlākiya"

DenisNosnitsin1970 replied:
Za-amlakiya, three verses.

CarstenHoffmannMarburg replied:
> Za-amlakiya, three verses.

I do not understand this comment. Do you want to see the transcription updated from za-ʾamlākiya to Za-amlakiya?

DenisNosnitsin1970 replied:
That is ok, I only confirmed what Jonas said. No change needed.

karljonaskarlsson reviewed (approved):
Looks good!

CarstenHoffmannMarburg commented on new/LIT14520QeneZawahabkani.xml:
```diff
@@ -0,0 +1,67 @@
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="EMIP00499"/>, fol. 160b. Division into verses is indicated with ። .</note>
```
It should be 160r according to @DenisNosnitsin1970. I cannot check at the moment, because the server is offline. I think, it is on the third column. So 160rc would be perfect in this case.

CarstenHoffmannMarburg replied:
The folio is infact 157rb. I have checked it today.

eu-genia reviewed (changes requested):
Please never create pull requests for records with numbers that are not in sequence. I have cleaned the backend and removed this file from the app. please create a new file with a new number and replace the file in this pull request, do not merge the current file

eu-genia commented:
@karljonaskarlsson the same applies to your file created as 
![image](https://github.com/user-attachments/assets/137f94c0-39fb-4f12-bbb5-da3b89726776)

I am sorry I cannot be constantly online when I am on vacation but please never merge files with wrong numbers, thank you

CarstenHoffmannMarburg commented:
> Please never create pull requests for records with numbers that are not in sequence. I have cleaned the backend and removed this file from the app. please create a new file with a new number and replace the file in this pull request, do not merge the current file

OK! Sorry for that!

CarstenHoffmannMarburg commented:
> Please never create pull requests for records with numbers that are not in sequence. I have cleaned the backend and removed this file from the app. please create a new file with a new number and replace the file in this pull request, do not merge the current file

I newly created LIT7319QeneZawahab. I will push in a minute.

DenisNosnitsin1970 commented on new/LIT7319QeneZawahab.xml:
```diff
@@ -0,0 +1,67 @@
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዘወሀብከኒ፡ ግብረ፡ አምጣነ፡ በትጋህ፡ ፈጸምኩ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Za-wahabkanni gǝbra ʾamṭāna ba-tǝgāh faṣṣamku ... (qǝne of the za-ʾamlākiya-type)</title>
+                <title xml:lang="en" corresp="#t1">I completed the task of the vigil you gave to me ...</title>
```
I would add "since". Since I diligently completed the task you gave me...

(I wrote, while he was subjecting me to the hardship, 
A piece of cloth, Gorgoryos, please offer me as gift)

CarstenHoffmannMarburg replied:
Ok. I will add a translation of the poem. Is it not better to translate "... I have sent, ..., a piece of cloth,...)

eu-genia reviewed (approved):
thank you!
